.golangci.yml

```diff
@@ -93,41 +99,23 @@ issues:
   max-issues-per-linter: 0
   max-same-issues: 0
   exclude-dirs: [node_modules, public, web_src]
+  exclude-case-sensitive: true
   exclude-rules:
-    # Exclude some linters from running on tests files.
     - path: _test\.go
-      linters:
-        - gocyclo
-        - errcheck
-        - dupl
-        - gosec
-        - unparam
-        - staticcheck
+      linters: [gocyclo, errcheck, dupl, gosec, unparam, staticcheck]
```

Comment:
I like the old style which looks like more clear.

Reply:
Hmm, I much prefer the compact variant.

Reply:
Change it back.

Reply:
Thanks for changing it back. A more concrete reason to list it as it is currently is to make differences easier to read when adding/removing an item from this list.